content/docs/2.10/scalers/apache-kafka.md

```diff
@@ -73,20 +73,21 @@ partition will be scaled to zero. See the [discussion](https://github.com/kedaco
 ### Authentication Parameters
 
  You can use `TriggerAuthentication` CRD to configure the authenticate by providing `sasl`, `username` and `password`, in case your Kafka cluster has SASL authentication turned on. If you are using SASL/OAuthbearer you will need to provide `oauthTokenEndpointUri` and `scopes` as required by your OAuth2 provider. If TLS is required you should set `tls` to `enable`. If required for your Kafka configuration, you may also provide a `ca`, `cert`, `key` and `keyPassword`. `cert` and `key` must be specified together.
+ Another alternative is to specify in ScaledObject `enableTls` and `saslAuthType`  instead of `sasl` and `tls` in TriggerAuthentication respectively.
 
 **Credential based authentication:**
 
 **SASL:**
 
-- `sasl` - Kafka SASL auth mode. (Values: `plaintext`, `scram_sha256`, `scram_sha512`, `oauthbearer` or `none`, Default: `none`, Optional)
+- `sasl` / `saslAuthType` - Kafka SASL auth mode. (Values: `plaintext`, `scram_sha256`, `scram_sha512`, `oauthbearer` or `none`, Default: `none`, Optional)
```

Comment:
I think that we can reuse the same key as it can be recovered from both places

Reply:
Yes thats what I have in mind too as I start writing the doc 😂. Should be good to reuse the same key to avoid further confusion